internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -482,8 +482,8 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	if err := CheckRegionConfigsPriorityOrder(params.GetReplicationSpecs()); err != nil {
 		return diag.FromErr(err)
 	}
-	// cannot call latest API (2024-10-23 or newer) as it can enable ISS autoscaling
-	cluster, _, err := connV220240805.ClustersApi.CreateCluster(ctx, projectID, params).Execute()
+
+	cluster, _, err := connV2.ClustersApi.CreateCluster(ctx, projectID, params).Execute()
```

Comment:
Noting that create operation has been adjusted to use the latest SDK to avoid requiring all `param` related functions to be repeated for SDK version `2024-08-05` which would have to be shortly removed by [CLOUDP-285705](https://jira.mongodb.org/browse/CLOUDP-285705). [CLOUDP-285705](https://jira.mongodb.org/browse/CLOUDP-285705) will further adjust the create operation and cover manual testing of creation behaviour, (which the work of this PR did not cover).